core/kmod/sn_netdev.c

```diff
@@ -610,6 +610,10 @@ static u16 sn_select_queue(struct net_device *netdev,
 			   struct sk_buff *skb,
 			   void *accel_priv,
 			   select_queue_fallback_t fallback)
+#elif LINUX_VERSION_CODE >= KERNEL_VERSION(5,2,0)
```

Comment:
(million years late, sorry)

IMO the current order is somewhat hard to follow. Perhaps this order would be better?

```
...
#elif version < 4.18.0
   ...
#elif version < 5.2.0
   ...
#else 
   ...
#endif